app/main/views/communications.py

```diff
@@ -43,6 +43,24 @@ def manage_communications(framework_slug):
     )
 
 
+@main.route('/communications/<framework_slug>/files/<string:comm_type>/<path:filepath>', methods=['GET'])
+@role_required('admin-framework-manager')
+def download_communication(framework_slug, comm_type, filepath):
+    if comm_type not in _comm_types:
+        abort(404)
+
+    # ensure this is a real framework
+    data_api_client.get_framework(framework_slug)
```

Comment:
Could we use `app.helpers.frameworks.get_framework_or_404` here?

Reply:
Oh yeah